### ISSUE TITLE
Migration to landscape2

### ISSUE THREAD

**Issue by tegioz** [lfai/lfai-landscape#678]
Hi! 👋

We've been working on a new version of the landscape generating software, [landscape2](https://github.com/cncf/landscape2). This new version relies on the same `landscape.yml` file as the main data source, so nothing changes regarding adding or updating entries in the landscape.

The [CNCF landscape](https://landscape.cncf.io/) has already been successfully migrated to landscape2, and we'd like to start migrating more landscapes 🙂

We are already building the LF AI & DATA landscape using landscape2, you can check it out at <https://lfai.landscape2.io/>. It'd be great if you could give it a try when you have a chance and, whenever you are ready, we can proceed with the migration.

The process is very simple and would involve the following steps:

- [ ] Add/update some DNS entries [1]
- [ ] Stop background automated processes that write to this repository (i.e. CNCF bot)
- [ ] Unlink this repository from Netlify
- [ ] Update README file and clean up some files no longer needed in this repository
- [ ] Setup the [landscape2-validate-action](https://github.com/cncf/landscape2-validate-action) to validate the `landscape.yml` file on PRs (optional)

We can take care of the last 4 points, you'd only need to add/update some DNS entries.

Landscape2 supports embedding a view with items in a category/subcategory in another website, but it works slightly different than landscape1. If parts of this landscape are embedded using this mechanism on other websites, we'd appreciate if you could let them know about [how it works now](https://github.com/cncf/landscape2#embeddable-views).

Please ping me if you have any question 😉

[1] We’d need to know the hostnames you'd like to use to prepare our end to listen on them before switching the DNS (i.e. [landscape.lfai.foundation](http://landscape.openssf.org/)). Once we are ready we'll provide you with the destination for the DNS entries so that you can update them.

**Comment by ibrahimhaddad** [comment]
@tegioz Thanks Sergio, I submitted an internal ticket for the CNAME entries. I will drop you a note once this is done. Thx.

**Comment by ibrahimhaddad** [comment]
@tegioz the records have been added. Thanks.

**Comment by tegioz** [comment]
Thanks @ibrahimhaddad, but they don't seem to be ready yet:

```
% host -t cname _6b19a9dfa414c9a7c8ceee3064f0d871.landscape.lfai.foundation.
Host _6b19a9dfa414c9a7c8ceee3064f0d871.landscape.lfai.foundation. not found: 3(NXDOMAIN)
% host -t cname _ef6fbaebd04f81c3dc5b0fdee6a1b13d.l.lfai.foundation.
Host _ef6fbaebd04f81c3dc5b0fdee6a1b13d.l.lfai.foundation. not found: 3(NXDOMAIN)
```

**Comment by ibrahimhaddad** [comment]
Give it some more time? It's done. It may be caching or it needs some time to propagate. @tegioz

**Comment by ibrahimhaddad** [comment]
@tegioz any update on this? Thanks.

**Comment by tegioz** [comment]
The DNS entries are not ready yet @ibrahimhaddad. You can check them yourself with the command I shared a few comments above. New entries are usually available pretty quick, so I suspect something went wrong when they were added.

Please let me know when they're ready and we'll continue with the process 🙂
